src/test/OpenEXRTest/tmpDir.h

```diff
@@ -15,6 +15,6 @@
     #define rmdir _rmdir
 #else
     #include <sys/stat.h> // for mkdir
-    #define IMF_TMP_DIR "/var/tmp/"
+    #define IMF_TMP_DIR "/tmp/"
```

Comment:
Is this necessary? It should be OK, although space is often limited under /tmp, and tests may write large image files.

Reply:
Yes the change is unfortunately needed. I could do also this change:

```
    #ifndef IMF_TMP_DIR
        #define IMF_TMP_DIR "/var/tmp/"
    #endif    
```

This way a user can provide from outside the IMF_TMP_DIR via a define (similar as it is done already for `ILM_IMF_TEST_IMAGEDIR`) - would this be an option?

Reply:
Looking at the usage of IMF_TMP_DIR in main.cpp, wouldn't a better approach be to use mkdtemp() on linux/macOS and something based on GetTempPath() on Windows?

Reply:
`IMF_TMP_DIR` is duplicated at several places. It makes sense to refactor this. Maybe the cleanest solution is to refactor this using `mkdtemp` and `GetTempPath` and using this refactoring also as a chance for supporting hermetic Bazel tests.

Anyways, in the meanwhile, I have also an idea of how to run the tests with Bazel, without touching the source code (C/C++ code). Instead of using `cc_test` one can use simply `cc_binary` and use the CI job to report if it was failing or not.